x-pack/legacy/plugins/task_manager/server/task_store.test.ts

```diff
@@ -234,7 +234,7 @@ describe('TaskStore', () => {
 
       expect(args).toMatchObject({
         body: {
-          sort: [{ 'task.taskType': 'desc' }, { _id: 'desc' }],
```

Comment:
It looks like there is an `updated_at`. I wonder if that would work if sorting in a descending order if required.

```
{
    "_index" : ".kibana_task_manager_1",
    "_id" : "task:oss_telemetry-vis_telemetry",
    "_score" : 1.0,
    "_source" : {
        "migrationVersion" : {
            "task" : "7.6.0"
        },
        "task" : {
        "taskType" : "vis_telemetry",
        "retryAt" : null,
        "runAt" : "2020-01-10T08:00:00.000Z",
        "startedAt" : null,
        "state" : """{"runs":1}""",
        "params" : "{}",
        "ownerId" : null,
        "scheduledAt" : "2020-01-09T22:07:28.092Z",
        "attempts" : 1,
        "status" : "idle"
        },
        "references" : [ ],
        "updated_at" : "2020-01-09T22:07:31.278Z",
        "type" : "task"
    }
}
```

Reply:
Looking at the docs for `search_after` (which is what is used for pagination) 
 https://www.elastic.co/guide/en/elasticsearch/reference/current/search-request-body.html#request-body-search-search-after, it says

>  A field with one unique value per document should be used as the tiebreaker of the sort specification. Otherwise the sort order for documents that have the same sort values would be undefined and could lead to missing or duplicate results. The _id field has a unique value per document but it is not recommended to use it as a tiebreaker directly.

So it looks like we're using _id for this tiebreaking, which isn't recommended anyway because there are no doc_values.  

The  recommendation is to duplicate the content of the _id.  Or we generate another unique value.  Not sure something like `updated_at` would be close enough to unique.

Reply:
Ya, looking at the [`mappings.json`](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/task_manager/server/mappings.json), I didn't see anything stable enough to use as an alt-stabilizer.  `updated_at` would probably be pretty good, but it's not part of the TM mappings, I think part of the base SO mappings - I guess that's probably good enough.

I'm hoping that we're not really dependent on the ordering here, but if we are, `updated_at` might be a good replacement ...